lexicons/com/atproto/resource/authentication.json

```diff
@@ -0,0 +1,24 @@
+{
+  "lexicon": 1,
+  "id": "com.atproto.resource.authentication",
+  "defs": {
+    "main": {
+      "type": "auth-resource",
+      "description": "Grants access to authentication for third party servcies (service auth)",
```

Comment:
Should we forbid the use of `aud=*&lxm=*`?

Reply:
I don't necessarily think so. We have resources that are more powerful (for instance full write access to your repo I would consider more powerful). I think we can just flag something like that pretty aggressively in the auth screen